api/external/applications/applications.proto

```diff
@@ -128,7 +235,11 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).resource = "retention:serviceGroups";
     option (chef.automate.api.iam.policy).action = "retention:serviceGroups:update";
   };
+  /*
+  GetVersion
 
+  Returns the current version of the applications-service
+  */
   rpc GetVersion (common.version.VersionInfoRequest) returns (common.version.VersionInfo) {
     option (google.api.http).get = "/beta/applications/version";
     option (chef.automate.api.policy).resource = "service_info:version";
```

Comment:
i think thee version endpoint should be hidden?
https://github.com/chef/automate/blob/master/components/automate-gateway/api/compliance/reporting/reporting.proto#L280